requirements.txt

```diff
@@ -37,6 +37,7 @@ drf-nested-routers==0.93.4
 drf-yasg==1.20.0
 environs==9.5.0
 gunicorn==20.1.0
+mdmail==0.1.3
```

Comment:
This project does not seem to be active. Is that an issue? Could we take the bits we need and add them to Hypha?

Reply:
I'll take a look.

Reply:
Done